style.css

```diff
@@ -0,0 +1,44 @@
+body {
+font-family: 'Shadows Into Light', cursive;
+background-color: hsl(208,30%,90%);
+}
+
+h1 {
+  font-size: 52px;
+  text-transform: capitalize;
+  font-weight: bold;
```

Comment:
This reiterates the  [default browser styles](https://browserdefaultstyles.com/) for headings -- they are bold by default -- and, therefore, can be removed:
```suggestion
```